### ISSUE TITLE
Attendance

### ISSUE THREAD

**Issue by praneeth-bojanala5** [therealsujitk/android-vtop-chennai#24]
## Feature Request

Show Attendance with dates

**Comment by pythongiant** [comment]
@therealsujitk are you working on this issue atm?

**Comment by therealsujitk** [comment]
> @therealsujitk are you working on this issue atm?

@pythongiant no, I am currently not working on this issue and probably won't work on it anytime soon.

> idm helping to contribute with this

That would be great! I'm still not sure how to go about implementing this, I don't think it'll be a good idea to extract several records of data for each and every course while syncing as that would slow down syncing by a significant amount, so I was thinking along the lines of signing in again and getting the data for individual courses when asked for by the user.

Either way it can be a pretty massive implementation :sweat_smile:

**Comment by adityajha2005** [comment]
is the issue still there?
maybe I can do it
